server_installation/topics/operator/monitoring-stack.adoc

```diff
@@ -0,0 +1,53 @@
+
+=== Viewing Operator Metrics
+
+To view metrics for the Operator, you can use the Grafana Dashboard and Prometheus Alerts from the {application_monitoring_operator}. For example, you can view metrics such as the number of controller runtime reconciliation loops, the reconcile loop time, and errors.
+
+==== Installing the {application_monitoring_operator}
+
+The {project_operator} integration with the {application_monitoring_operator} requires no action. You do not need to configure integration with Grafana, but you must install the {application_monitoring_operator} in the cluster.
+
+.Prerequisites
+
+* The {project_operator} is installed.
+
+* You have cluster administrator permissions or an equivalent level of permissions granted by an administrator.
+
+.Procedure
+
+. Install the {application_monitoring_operator} by using the link:{application_monitoring_operator_installation_link}[documentation].
+
+. Annotate the namespace used for the {project_operator} installation. For example:
++
+[source,bash,subs=+attributes]
+----
+oc label namespace <namespace> monitoring-key=middleware
+----
+
+. Confirm monitoring is working by searching for Prometheus and Grafana route in the `application-monitoring` namespace.
++
+.Routes in OpenShift web console
+image:{project_images}/operator-application-monitoring-routes.png[]
+
+==== Using Grafana and Prometheus web UIs
+
+Grafana and Promotheus each provide graphical information about Operator activities.
+
+* The Operator installs a pre-defined Grafana Dashboard as shown here:
++
+.Grafana Dashboard
+image:{project_images}/operator-graphana-dashboard.png[]
++
+[NOTE]
+====
+If you make customizations, we recommend that you clone the Grafana Dashboard so that your changes are not overwritten during an upgrade.
+====
+
+* The Operator installs a set of pre-defined Prometheus Alerts as shown here:
++
+.Prometheus Alerts
+image:{project_images}/operator-prometheus-alerts.png[]
+
+.Additional resources
+
+For more information, see link:https://docs.openshift.com/container-platform/4.3/monitoring/cluster_monitoring/prometheus-alertmanager-and-grafana.html[Accessing Prometheus, Alertmanager, and Grafana].
```

Comment:
Latest document is 4.4: https://docs.openshift.com/container-platform/4.4/monitoring/cluster_monitoring/prometheus-alertmanager-and-grafana.html. I suggest to change the version number with the "latest" word.